webclient/services/mailer.py

```diff
@@ -55,6 +55,7 @@
     "E-P-03a",
     "E-P-03b",
     "E-P-07",
+    "E-P-08",
```

Comment:
@motyc mam udelat rucne sql na pridani do notifikace_typ, nebo to necham an vas?

Reply:
@jnihnat Doplněno na všechny servery pomocí `INSERT INTO notifikace_typ(ident_cely) VALUES ('E-P-08');`